security/ThreatModel.md

```diff
@@ -0,0 +1,21 @@
+### Threat Model
+
+To: important@someemail.com
+
+From: patrick
+
+Subject: Security Threat
+
+After looking at our newly rolled out app for our loan service, it seems that personal
+user information is not getting stored properly. If I look at the code more closely, it 
+turns out that client info and passwords are exposed when they should be hashed at the very least. 
+Potential adversaries to this would be identify thiefs and quite frankly anyone looking to make a quick buck.
+I advise that the app be shut down until we can identify the root cause for the issue and fix the problem so 
+that user info is not continuing to be exposed.
+
+Furthermore we need to reach out to all users and have them reset their passwords and usernames to mitigatre the risk. 
+I would also recommend that we prompt user to install 2 factor authentication which further mitigates any
```

Comment:
Absolutely! If you were to identify a threat like this, all users should be notified right away and made to change their passwords and enable any other security features until you can be sure that their data is safe.